assets/src/edit-story/masks/display.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+
+import StoryPropTypes from '../types';
+import { getElementMask } from './';
+
+const FILL_STYLE = {
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  right: 0,
+  bottom: 0,
+};
+
+export default function WithMask({
+  element,
+  fill,
+  style,
+  children,
+  box,
+  ...rest
+}) {
+  const mask = getElementMask(element);
+
+  if (!mask?.type) {
+    return children;
```

Comment:
I'm pretty sure we still need a `div` here. This is where we planned to place layer opacity and number of other properties that are meaningful to a mask, even if the mask is a simple div rectangle. Basically, as soon as we explicitly accept `style` above, we should use it.